### ISSUE TITLE
Sometimes not possible to save a new file

### ISSUE THREAD

**Issue by BoBBer446** [k0shk0sh/FastHub#1560]
FastHub Version: 4.6.3
Android Version: 7.0 (SDK: 24)
Device Information:

Manufacturer: samsung
Brand: samsung
Model: Galaxy Tab A 10.1

![screenshot_20180226-091530](https://user-images.githubusercontent.com/10163536/36659856-2ed73166-1ad6-11e8-9219-4797fcb3256b.png)

I delete all brunches to work with only one brunch. Now i create a file , write a commit and give the file a name. but if i click top right to push i get an issue 
**Addons not a file**

Yes, "Addons" is the sub Folder from main script.

it works if i cloes in tast manager and start again.

**Comment by BoBBer446** [comment]
ah okay, now it dosent work with close app in tastker and reopen.
igo into the sub folder "addons" and click + to create file.
![screenshot_20180226-092610](https://user-images.githubusercontent.com/10163536/36660113-393e04c6-1ad7-11e8-9a57-ca6c28d6fc93.png)
pre filled file name calls "addons" and it seems that the new file name is not accept

**Comment by BoBBer446** [comment]
Same in Note 4 Android 7.1.1 lineage

**Comment by k0shk0sh** [comment]
im abit confused as I tested this and it works fine, could u create a dummy repo that u test on? 

Also if u still have the same prob, link me to that repo so I could test it there myself.

**Comment by BoBBer446** [comment]
I try to create a file in Ajax folder in my repo eBay community edition.

But i get the same issue
![](https://i.imgur.com/zQZyyzI.png)

**Comment by k0shk0sh** [comment]
Is it a repo in organization?

**Comment by BoBBer446** [comment]
No.i create a test repo but the same issue![](https://i.imgur.com/UQWeHiw.png)

**Comment by k0shk0sh** [comment]
Private or public?

**Comment by BoBBer446** [comment]
Public 
On note 4 trlte los 15.1 (Android 8.1) but on Galaxy tab A same issue.

Latest fasthub application installed

**Comment by BoBBer446** [comment]
I try again

I create repo on Website. I create folder with file.
Now i open fasthub and switch to repo
I go into folder
I press the "+" button and create file
I click save and geht issue

https://github.com/BoBBer446/testTWOFASTHUB
![screenshot_chrome_20180326-194137](https://user-images.githubusercontent.com/10163536/37922896-1d7afd62-312e-11e8-815a-80841cd076a4.png)
![screenshot_chrome_20180326-194214](https://user-images.githubusercontent.com/10163536/37922912-2977b4ac-312e-11e8-8681-0837a89d3406.png)
![screenshot_fasthub_20180326-194243](https://user-images.githubusercontent.com/10163536/37922926-3445871a-312e-11e8-83bd-fd61caacc726.png)
![screenshot_fasthub_20180326-194243](https://user-images.githubusercontent.com/10163536/37922944-3caca12c-312e-11e8-905b-a19b30a2dc51.png)
![screenshot_fasthub_20180326-194246](https://user-images.githubusercontent.com/10163536/37922957-4280cb78-312e-11e8-8f93-e0ee614c1401.png)
![screenshot_fasthub_20180326-194251](https://user-images.githubusercontent.com/10163536/37922969-4856b42c-312e-11e8-8858-72f928e0cac1.png)
![screenshot_fasthub_20180326-194342](https://user-images.githubusercontent.com/10163536/37922982-4eb6aa98-312e-11e8-99bd-d624ba4fb8f8.png)

**Comment by emrahyumuk** [comment]
Same issue. There is a problem when i am creating new file in a folder. 

\<folderName\> is not a file.

FastHub Version: 4.6.7
Android Version: 6.0.1
Manufacturer: Sony
Brand: Sony
Model: Xperia Z3

**Comment by Eurobertics** [comment]
Bumb...
Same here:
Try to add add a file in a public repo: /src/Models
Filename field is prefilled with "Models".
Added code, commit message and changed filename field to filename.

Response: "src/Models is not a file"

Tested on mobile and tablet.

FastHub Version: 4.6.7
Android Version: 9.0.0
Manufacturer: Samsung
Brand: Samsung
Model: Samsung Galax Note 9 / Samsung Galax Tab S4

**Comment by cinocode** [comment]
So I can reproducibly not create a file in any subfolder (only at the application root) and also I see no workaround (moving a file or similar). At first I thought I just hadn't figured out how to use the app properly. Maybe I had a wrong file name format.

Then I check for other users with similar problems and I find this issue where the problem is well documented with steps to reproduce since a year and a half. Creating a file in a subfolders seems pretty essential to me. Any intention of working on this?
